maven-core/src/main/java/org/apache/maven/project/collector/MultiModuleCollectionStrategy.java

```diff
@@ -91,7 +91,7 @@ public MultiModuleCollectionStrategy( ModelLocator modelLocator, ProjectsSelecto
 
             if ( fallThrough )
             {
-                logger.debug( "Multi module project collection failed: {}"
+                LOGGER.debug( "Multi module project collection failed:{}"
```

Comment:
Please restore the space after the colon.

Reply:
Same here